dev-requirements.txt

```diff
@@ -2,3 +2,4 @@ pytest==3.3.1
 pytest-cache==1.0
 pytest-cov==2.5.1
 pytest-pep8==1.0.6
+-e git+git@github.com:dankolbman/eralchemy.git@1058c12b828013144f861cb59f0e2c280a1a5ef6#egg=eralchemy
```

Comment:
Waiting for [this](https://github.com/Alexis-benoist/eralchemy/pull/42) to merge.